newrelic.ini

```diff
@@ -0,0 +1,210 @@
+# ---------------------------------------------------------------------------
+
+#
+# This file configures the New Relic Python Agent.
+#
+# The path to the configuration file should be supplied to the function
+# newrelic.agent.initialize() when the agent is being initialized.
+#
+# The configuration file follows a structure similar to what you would
+# find for Microsoft Windows INI files. For further information on the
+# configuration file format see the Python ConfigParser documentation at:
+#
+#    http://docs.python.org/library/configparser.html
+#
+# For further discussion on the behaviour of the Python agent that can
+# be configured via this configuration file see:
+#
+#    http://newrelic.com/docs/python/python-agent-configuration
+#
+
+# ---------------------------------------------------------------------------
+
+# Here are the settings that are common to all environments.
+
+[newrelic]
+
+# You must specify the license key associated with your New
+# Relic account. This key binds the Python Agent's data to your
+# account in the New Relic service.
+license_key = 27ab02903d0967548efee1eec38b13a76c24NRAL

```

Comment:
```suggestion
```
Since New Relic [recommends](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#ingest-recommendations) keeping the license key secret, it might be a good idea to omit it here and maybe change it to a new one. 
If I understand [this doc](https://docs.newrelic.com/docs/apm/agents/python-agent/configuration/python-agent-configuration/) correctly, we shouldn't need to include the key here since we have it set as an environment variable.

Reply:
I was concerned about putting this in the github branch but I was unaware whether the environment variable was sufficient for the Python Agent as well as the New Relic APM. Since it seems like you are correct based upon those links you have provided I would agree in removing this section of the newrelic.ini file.

Reply:
I also just updated the license key that is stored in the Heroku environment variables.